Add support for TS-7553-V2 -N model variant

ts-kris opened the pull request:
Rewrote and commented check for rev C and -N variants

ts-kris committed:
technologic: ts7753v2: Reworked pcbrev logic

This is to further impact FDT loading process in order
to better accommodate custom option variants.

The main boot scripts were also modified to attempt to boot
the specified pcbrev FDT file. If not present, then fall back
to the default file. Future custom options which may or may not
require a custom FDT file will both benefit from this with no
future U-Boot changes.

Additionally added a silopresent override from environment
variables.

luckshot reviewed (approved):
This way is probably cleaner.  Looks good to me.

ts-kris committed:
build-imx6ul: Add TS-7553-V2 to script

ts-kris committed:
ts7553v2: Changed "pcbrev" use to "variant"

More descriptive and less misleading this way

ts-kris committed:
technologic: ts7553v2: Clear "variant" from env if unused

This will prevent issues if this variable is set and saved.
Setting a var to "" will effectively remove it from the env.
